src/app/clusters/scenes/scenes.cpp

```diff
@@ -1054,11 +1054,11 @@ bool emberAfPluginScenesServerParseViewScene(const EmberAfClusterCommand * cmd,
             emberAfPutInt16uInResp(ZCL_THERMOSTAT_CLUSTER_ID);
             length = &appResponseData[appResponseLength];
             emberAfPutInt8uInResp(0); // temporary length
-            emberAfPutInt16uInResp(entry.occupiedCoolingSetpointValue);
+            emberAfPutInt16uInResp(static_cast<uint16_t>(entry.occupiedCoolingSetpointValue));
```

Comment:
I would much prefer it if we defined a `emberAfPutInt16InResp` that takes a signed `int16_t` and does the casting under the hood.  That will make it much simpler to change encodings later as needed.